frontend/src/pages/Home.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import StatusBubbles from '../components/StatusBubbles/StatusBubbles';
+import TableComponent from '../components/Table/Table';
+
+// Function to generate column names and data
+const createTableData = (columns, rows) => {
+  return {
+    columns,
+    data: rows
+  };
+};
+
+function Home() {
+  // State to track which table is currently visible
+  const [currentTable, setCurrentTable] = useState('attentionRequired'); // Default to first table
```

Comment:
instead of using currentTable, we can keep selected tab as there can be additional data specific to a tab

Reply:
updated the same in this discussion 

https://github.com/ajaychinni/NextGenHr/pull/1#discussion_r1801254862